spark-atlas-connector/src/test/scala/org/apache/spark/sql/kafka010/atlas/KafkaHarvesterSuite.scala

```diff
@@ -77,145 +63,6 @@ class KafkaHarvesterSuite extends StreamTest {
     assert(KafkaHarvester.extractTopic(microBatchWriter) === None)
   }
 
-  private class AtlasQueryExecutionListener extends QueryExecutionListener {
-    private val executionId = new AtomicLong(0L)
-    val queryDetails = new mutable.MutableList[QueryDetail]()
-
-    override def onSuccess(funcName: String, qe: QueryExecution, durationNs: Long): Unit = {
-      queryDetails += QueryDetail(qe, executionId.getAndIncrement(), durationNs)
-    }
-
-    override def onFailure(funcName: String, qe: QueryExecution, exception: Exception): Unit = {
-      fail(exception)
-    }
-  }
-
-  test("Streaming query with Kafka source and sink") {
```

Comment:
I just removed this test since it becomes redundant.

Reply:
Please don't remove anything in this stage.

Reply:
The new test is actually verifying the same thing (the new test covers broader code path), so the code is merely duplicated. Would we leave this as it as with allowing code duplication, or with cleaning up code dup.?

Reply:
No. Please don't remove anything.

Reply:
OK. I've restored the existed test back.